zipline/finance/trading.py

```diff
@@ -93,13 +92,34 @@ def __init__(
         max_date=None,
         env_trading_calendar=tradingcalendar
     ):
+        # `tc_tdays` is short for "trading calendar trading days"
+        # `tc_tday` is short for "trading calendar trading day"
+        tc_tdays = env_trading_calendar.trading_days
+        tc_tday = env_trading_calendar.trading_day
```

Comment:
There may have been some improvements to the how to define holidays in Pandas, but I haven't caught up completely on it.

https://github.com/pydata/pandas/issues/7070